package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "sql-ddl-to-json-schema",
-  "version": "4.1.1",
+  "version": "4.2.0",
```

these are breaking changes. I agree it should be 5.0.0
```suggestion
  "version": "5.0.0",
```

---

ok, a version commit has been pushed, please check it out